
string target_location
float64[] target_coord
string[] options

---

bool result
